modules/liveIntentAnalyticsAdapter.js

```diff
@@ -123,19 +120,15 @@ function ignoreUndefined(data) {
 
 let liAnalytics = Object.assign(adapter({URL, ANALYTICS_TYPE}), {
   track({ eventType, args }) {
-    if (eventType == AUCTION_END && args && isSampled) { handleAuctionEnd(args); }
+    if (eventType == AUCTION_END && args) { handleAuctionEnd(args); }
   }
 });
 
 // save the base class function
 liAnalytics.originEnableAnalytics = liAnalytics.enableAnalytics;
-
 // override enableAnalytics so we can get access to the config passed in from the page
 liAnalytics.enableAnalytics = function (config) {
-  initOptions = config.options;
-  const sampling = (initOptions && initOptions.sampling) ?? DEFAULT_SAMPLING;
-  isSampled = Math.random() < parseFloat(sampling);
-  bidWonTimeout = (initOptions && initOptions.bidWonTimeout) ?? DEFAULT_BID_WON_TIMEOUT;
+  bidWonTimeout = config?.options?.bidWonTimeout ?? DEFAULT_BID_WON_TIMEOUT;
   liAnalytics.originEnableAnalytics(config); // call the base class function
```

Comment:
As far as I can see, the base `enableAnalytics` already handles sampling. There is not need to sample in the `track` function above.